public/static/locales/cy/common.json

```diff
@@ -352,6 +352,7 @@
   "Your message is too long. Please shorten it to less than 1500 characters": "Mae eich neges yn rhy hir. A fyddech cystal â’i chwtogi i lai na 1500 nod",
   "Please select which days this establishment is open": "Dewiswch pa diwrnodau mae'r sefydliad hwn ar agor",
   "Please describe when this establishment is open": "Disgrifiwch pryd mae'r sefydliad hwn ar agor",
+  "Enter a valid establishment web address": "Rhowch gyfeiriad gwe sefydliad dilys",
```

Comment:
Move this to join the rest of the validation statements to keep them all together